pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -361,6 +361,12 @@ in
     buildInputs = [ which v8 python2 ];
     buildFlags = [ "--with-system-v8=true" ];
     dontBuild = false;
+    # The gem includes broken symlinks which are ignored during unpacking, but
+    # then fail during build. Since the content is missing anyway, touching the
+    # files is enough to unblock the build.
+    preBuild = ''
+      touch vendor/depot_tools/cbuildbot vendor/depot_tools/chrome_set_ver vendor/depot_tools/cros_sdk
```

Comment:
I'll wait for ofborg builds to verify all systems are ok with this. Without the touch, we get:
```
WARNING:  libv8-8.4.255.0 ships with a dangling symlink named vendor/depot_tools/cbuildbot pointing to missing vendor/depot_tools/cros file. Ignoring
WARNING:  libv8-8.4.255.0 ships with a dangling symlink named vendor/depot_tools/chrome_set_ver pointing to missing vendor/depot_tools/cros file. Ignoring
WARNING:  libv8-8.4.255.0 ships with a dangling symlink named vendor/depot_tools/cros_sdk pointing to missing vendor/depot_tools/cros file. Ignoring
```
and then
```
ERROR:  While executing gem ... (Gem::InvalidSpecificationException)
    ["vendor/depot_tools/cbuildbot", "vendor/depot_tools/chrome_set_ver", "vendor/depot_tools/cros_sdk"] are not files
```